static/index.html

```diff
@@ -58,6 +59,7 @@ <h2>Queue batches of lightning channels to open in a single transaction</h2>
 				</div>
 				<output id="queued" class="invisible">
 					<h2>PayJoin here to open these channels</h2>
+					<code style="text-align: center;">⚠ This software is still extremely experimental and has not been vetted, use at your own risk ⚠</code>
```

Comment:
this is not code. best to save semantic elements for semantics and use styles to apply styles

Reply:
True lazy mans HTML 😅

Will resolve

Reply:
isn't it <code> just `font-family: monospace` ?

Reply:
It adds border and some other cool styling (put important bits in `.warning` css class)